src/commands/synthetics/compatibility.ts

```diff
@@ -50,7 +50,7 @@ export const replacePollingTimeoutWithBatchTimeout = (
 ): number | undefined => {
   // At this point, `global` should already have been moved to `defaultTestOverrides`
   const pollingTimeout = pollingTimeoutCliParam ?? config.defaultTestOverrides?.pollingTimeout ?? config.pollingTimeout
-  const isPollingTimeoutUsed = pollingTimeout !== DEFAULT_POLLING_TIMEOUT
+  const isPollingTimeoutUsed = pollingTimeout !== undefined && pollingTimeout !== DEFAULT_POLLING_TIMEOUT
```

Comment:
Is this change related to the selective re-run rate limiting or just something you noticed needing fixing?

Reply:
It's unrelated 😬 I noticed this when updating the unit tests on that PR - it shouldn't possible in the real world because `config.pollingTimeout` always defaults to `DEFAULT_POLLING_TIMEOUT` but we can keep this extra safety check